api/bucketsd/service.go

```diff
@@ -92,6 +92,7 @@ var (
 		FastRetrieval:   true,
 		DealStartOffset: 72 * 60 * 60 / powUtil.EpochDurationSeconds, // 72hs
 		MaxPrice:        100_000_000_000,
+		VerifiedDeal:    false,
```

Comment:
In these cases, I prefer to set explicitly false (even if that's the case as it's bool default), to be completely clear that we didn't miss to configure this.